### PR TITLE
Remove `WaitService`

### PR DESCRIPTION
**Pull request by dougbu**
- use `MvcTestFixture` instead of `TestHelper`
  - part of #3139
- continue to test e2e interaction of `FlushAsync()` with sections, partials, et cetera
- remove `FlushPointTest.FlushBeforeCallingLayout()`
  - `RazorPageTest.FlushAsync_ThrowsIfTheLayoutHasBeenSet()` unit test covers the scenario

### CODE REVIEW

File: test/Microsoft.AspNet.Mvc.FunctionalTests/FlushPointTest.cs [hunk 1]
```diff
@@ -92,112 +60,44 @@ public class FlushPointTest
...
-                GetTrimmedString(stream),
-                ignoreLineEndingDifferences: true);
-            waitService.WaitForServer();
-
-            // Assert - 2
-            Assert.Equal("<span>Nested content that takes time to produce</span>", GetTrimmedString(stream));
-        }
 
-        [Fact]
-        public async Task FlushBeforeCallingLayout()
```
Comment (pranavkm) [inline test/Microsoft.AspNet.Mvc.FunctionalTests/FlushPointTest.cs]: This is covered by a unit test - https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNet.Mvc.Razor.Test/RazorViewTest.cs#L1077

**Comment by pranavkm** [comment]
:shipit:

File: test/Microsoft.AspNet.Mvc.FunctionalTests/FlushPointTest.cs (hunk 1 above)
Comment (dougbu) [inline test/Microsoft.AspNet.Mvc.FunctionalTests/FlushPointTest.cs]: That's why I removed it entirely and mentioned the unit test in the commit description.

**Comment by dougbu** [comment]
ff19702

**Pull request closed without merging**